R/filter_panel_api.R

```diff
@@ -62,7 +62,11 @@ FilterPanelAPI <- R6::R6Class( # nolint
     #'
     #' @return `NULL`
     set_filter_state = function(filter) {
-      private$filtered_data$set_filter_state(filter)
+      if (private$filtered_data$get_filter_turn()) {
+        private$filtered_data$set_filter_state(filter)
+      } else {
+        warning("Filter Panel is turn off so the action can not be applied with api.")
```

Comment:
"turned off"
Please apply it into other places as well

BUT: please also see my previous comment - maybe it's not needed at all.

Reply:
obsolate